### ISSUE TITLE
C# Expression-Bodied Properties reveals underline implementation

### ISSUE THREAD

**Issue by peval27** [doxygen/doxygen#7550]
I've spotted that when writing something like:

```
/// <summary>
/// TCP port
/// </summary>
public int TcpPort => DefaultTcpPort;
```
the generated documentation is

> int Namespace.ClassName.TcpPort = >DefaultTcpPort

note the space between `=` and `>`.

This is not acceptable as it reveals an implementation detail to the public.

Used latest doxygen 1.8.17

**Comment by albert-github** [comment]
I don't see the space between the `=` and `>` in the HTML output, but I don't have the complete example just the lines you gave, so:
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.

Maybe also have a look at the configuration setting `MAX_INITIALIZER_LINES`.

**Comment by sgrigalunas** [comment]
I think this issue is with the Doxygen bug related to C# expression-bodied member properties. 

This has been reported in:
https://github.com/doxygen/doxygen/issues/6278
https://github.com/doxygen/doxygen/issues/6064

(not sure about the relevance of the space for this issue - but this bug exposes elements of the source code which may be the 'implementation detail' he is referring to)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7991 for the generatl issue, the space between the `=` and `>` I never saw.

**Comment by albert-github** [comment]
Code of #7991 has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

In case the current master (1.9.0 (9cf80a9)) does not solve the issue:
- please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.
- describe what is still going wrong.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
